packages/functional-tests/pages/login.ts

```diff
@@ -450,12 +437,12 @@ export class LoginPage extends BaseLayout {
     return this.page.locator(selectors.SUBMIT).click();
   }
 
-  async isCWTSPageHeader() {
-    return this.page.locator(selectors.CWTS_PAGE_HEADER).isVisible();
+  isCWTSPageHeader() {
+    return this.page.locator(selectors.CWTS_PAGE_HEADER);
   }
```

Comment:
When converting locators I suggest to consider making them accessors, updating their names and to use the [recommended built-in locators prioritizing user-facing attributes and explicit contracts](https://playwright.dev/docs/locators#quick-guide).

For example, I think this might be equivalent to

```suggestion
  get CWTSPageHeader() {
    return this.page.getByRole('heading', {name: 'Choose what to sync:'});
  }
```

and in practice this would look like:
```
await expect(login.CWTSPageHeader).toBeVisible();
```
instead of 
```
await expect(login.isCWTSPageHeader()).toBeVisible();
```
which makes more syntactical sense imo

Reply:
Thanks for the suggestion, I will make the changes.